server/historian/packages/historian-base/src/services/restGitService.ts

```diff
@@ -207,10 +207,11 @@ export class RestGitService {
         return this.post(`/repos/${this.getRepoPath()}/git/refs`, params);
     }
 
-    public async createSummary(summaryParams: IWholeSummaryPayload): Promise<IWriteSummaryResponse> {
+    public async createSummary(summaryParams: IWholeSummaryPayload, initial?: boolean): Promise<IWriteSummaryResponse> {
         const summaryResponse = await this.post<IWholeFlatSummary | IWriteSummaryResponse>(
             `/repos/${this.getRepoPath()}/git/summaries`,
-            summaryParams);
+            summaryParams,
+            initial !== undefined ? { initial } : undefined);
```

Comment:
would this not work?
```ts
initial ? { initial} : undefined
```

Reply:
`initial` can be `false` (as a boolean). If it's explicitly defined in the params, I want to make sure it's passed through, `false` or `true`. If it's not explicitly defined, I wanted it to remain `undefined`. `initial ? { initial } : undefined` would return undefined when `initial == false`, hence the explicit undefined check.

Functionally it doesn't really make a difference, but it felt better to make sure `undefined` was still supported and respected for back-compat.

Reply:
Makes sense!